packages/xo-web/src/xo-app/new-vm/index.js

```diff
@@ -1855,6 +1858,21 @@ export default class NewVm extends BaseComponent {
               </Item>
             </SectionContent>
           ),
+          isAdmin && isHvm && (
+            <SectionContent>
+              <Item>
+                <input
+                  checked={copyHostBiosStrings}
+                  className='form-control'
+                  disabled={affinityHost == null}
+                  onChange={this._linkState('copyHostBiosStrings')}
+                  type='checkbox'
+                />
+                &nbsp;
+                {_('newVmCopyHostBiosStrings')}
```

Comment:
On clicking on the label, the `checkbox` value will not be changed. 

You can use `label` tag with `htmlFor` property.

```html
<input id=ID /> <label htmlFor=ID></label>
```